randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -116,9 +116,15 @@ Extra - room_id: [3]
           # Upper way
           Screw Attack
   > Other to Entrance Hub
```

Comment:
Nocturnal Access -> vault is missing the same treatment here i think?

Reply:
Yup, forgot to check this one - fixed